### PR TITLE
Update microsoft-365-network-connectivity-principles.md

### PR DESCRIPTION
**Pull request by VargheseSivi**
Updates to Microsoft 365 endpoint category guidance.

### CODE REVIEW

**Comment by kelleyvice-msft** [comment]
Waiting for pandrew1 review before merge.

File: microsoft-365/enterprise/microsoft-365-network-connectivity-principles.md [hunk 1]
```diff
@@ -143,52 +141,35 @@ Microsoft is transparent about datacenter security, operational security, and ri
...
 
-    Optimization methods include:
+### Additional network considerations
+When optimizing connectivity to Microsoft 365, certain network configurations may have a negative impact on Microsoft 365 availability, interoperability, performance, and user experience. Microsoft has not tested the following network scenarios with our services, and they are known to cause connectivity issues.
+  - TLS termination or deep packet inspection of any M365 domains with customer proxies or other types of network devices or services [(Use third-party network devices or solutions with Microsoft 365 - Microsoft 365 | Microsoft Learn)](/troubleshoot/miscellaneous/office-365-third-party-network-devices).
+  - Blocking specific protocols or protocol versions such as QUIC, WebSocket’s, etc. by intermediate network infrastructure or service. 
+  - Forcing downgrade or failover of protocols (such as UDP  TCP, TLS1.3  TLS1.2  TLS1.1) used between client applications and Microsoft 365 services.
```
Comment (pandrew1) [inline microsoft-365/enterprise/microsoft-365-network-connectivity-principles.md]: Looks like there are some characters here which wont render on the page. Please confirm. "(such as UDP  TCP, TLS1.3  TLS1.2  TLS1.1)"

Comment (kelleyvice-msft) [reply microsoft-365/enterprise/microsoft-365-network-connectivity-principles.md]: Replacing unsupported characters (likely from Word) with -->

**Comment by kelleyvice-msft** [comment]
Sivi needs to review line 166 before merge

**Comment by VargheseSivi** [comment]
Hi Kelley,

I reviewed that line. It looks good. I don't have an option to approve it. Can you tell me what is expected?

Sivi

**Review by VargheseSivi — COMMENTED** [review]
the changes look good.

**Comment by garycentric** [comment]
@VargheseSivi @kelleyvice-msft 
Changes to _microsoft-365-network-connectivity-principles.md_ replaced the heading "New Office 365 endpoint categories" with "Optimizing connectivity to Microsoft 365 services", which broke links in the four articles shown in the [build report](https://github.com/MicrosoftDocs/microsoft-365-docs/pull/13502#issuecomment-2059776909). Please update links in other files when they're reported as broken by changes in the file that you've modified, and do not merge PRs with warnings.

**Pull request merged**